src/components/Person/Person.jsx

```diff
@@ -1 +1,18 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+    <p>
+      {person.age && (
+        <p className="Person__age">
+          {`I am ${person.age}`}
+        </p>
+      )}
+    </p>
```

Comment:
```suggestion
    {person.age && (
      <p className="Person__age">
        {`I am ${person.age}`}
      </p>
    )}
```
There is no need to create a container